src/components/ContinuationIn/AuthorizationProof.vue

```diff
@@ -277,12 +279,29 @@ export default class AuthorizationProof extends Mixins(DocumentMixin) {
         return // don't add to array
       }
 
-      // try to upload to Minio
-      let psu: PresignedUrlIF
+      // try to upload to Document Record Service
       try {
         this.isDocumentLoading = true
-        psu = await this.getPresignedUrl(file.name)
-        const res = await this.uploadToUrl(psu.preSignedUrl, file, psu.key, this.getKeycloakGuid)
+        const res = await this.uploadDocumentToDRS(
+          file,
+          this.DOCUMENT_TYPES.contInAuthorization.class,
+          this.DOCUMENT_TYPES.contInAuthorization.type,
+          this.getContinuationInConsumerDocumentId
+        )
+        // add file to array
+        this.authorization.files.push({
+          file: {
+            name: file.name,
+            lastModified: file.lastModified,
+            size: file.size
+          } as File,
+          fileKey: res.data.documentServiceId,
+          fileName: file.name
+        })
+
+        this.setContinuationConsumerDocumentId(res.data.consumerDocumentId)
+
+        this.isFileAdded = true
         if (!res || res.status !== StatusCodes.OK) throw new Error()
```

Comment:
Shouldn't you check this before adding the file to the array?

If you changed `uploadDocumentToDRS` to throw an exception on error then this code would be OK, because execution would go to the catch block and skip adding the file.

Reply:
It does make sense. I will upate it.